v20.2/schema-design-overview.md

```diff
@@ -83,7 +83,7 @@ For guidance on using temporary objects, see [Temporary Tables](temporary-tables
 
 ## Controlling access to objects
 
-CockroachDB supports both user-based and role-based access control. With roles, or with direct assignment, you can grant a [SQL user](authorization.html#sql-users) the [privileges](authorization.html#privileges) required to view, modify, and delete database schema objects.
+CockroachDB supports both user-based and role-based access control. With roles, or with direct assignment, you can grant a [SQL user](authorization.html#users) the [privileges](authorization.html#privileges) required to view, modify, and delete database schema objects.
```

Comment:
The hyperlinks should point to separate pages for user/role management and authorization.

Reply:
Including in the follow-up issue.